integrationtests/Paket.IntegrationTests/InstallSpecs.fs

```diff
@@ -354,21 +320,6 @@ let ``#1458 should not install conflicting deps from different groups``() =
     with
     | exn when exn.Message.Contains "Package Nancy is referenced in different versions" -> ()
 
-[<Test;Flaky>]
-let ``#2335 should install deps from different groups when using conditions``() =
```

Comment:
This is also a very interesting scenario we probably don't have covered in another test